spring-integration-zip/src/test/java/org/springframework/integration/zip/transformer/UnZipTransformerTests.java

```diff
@@ -45,40 +34,74 @@
 import org.springframework.messaging.MessagingException;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.zeroturnaround.zip.ZipException;
+import org.zeroturnaround.zip.ZipUtil;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.instanceOf;
 
 /**
- *
  * @author Gunnar Hillert
  * @author Artem Bilan
- *
  * @since 1.0
- *
  */
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration
 public class UnZipTransformerTests {
 
-	@Rule
-	public TemporaryFolder testFolder = new TemporaryFolder();
+    @Rule
+    public TemporaryFolder testFolder = new TemporaryFolder();
 
-	@Autowired
-	private ResourceLoader resourceLoader;
+    @Autowired
+    private ResourceLoader resourceLoader;
 
-	private File workDir;
+    private File workDir;
 
-	@Before
-	public void setup() throws IOException {
-		this.workDir = testFolder.newFolder();
-	}
+    @Before
+    public void setup() throws IOException {
+        this.workDir = testFolder.newFolder();
+    }
 
-	/**
```

Comment:
I will come back to review your test when you restore all other changes in this class.